.github/workflows/abtesting.yml

```diff
@@ -27,8 +27,6 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - uses: ruby/setup-ruby@359bebbc29cbe6c87da6bc9ea3bc930432750108
```

Comment:
Do we still need the `setup-ruby` with the commit hash?

Reply:
Hmm good question, testing in #9923

Reply:
So using the ruby/setup-ruby action is still needed: https://github.com/firebase/firebase-ios-sdk/runs/6943893771?check_suite_focus=true#step:4:18

But we can use `@v1` instead of the hash

Reply:
Resolved in a3a1bd0